### PR TITLE
new option in criticalKernelsThinning3D 

### PR DESCRIPTION
**Pull request by kerautret**
# PR Description
New option to use input image type to export result (allowing to keep same domain (and same image spacing when using ITK))

# Checklist

- [ ] Doxygen documentation of the code completed (classes, methods, types, members...).
- [ ] Main tool doxygen documentation (following existing documentation of [DGtalTools documentation](http://dgtal.org/doc/tools/nightly/).
- [ ] Check if it follows the tools structure described in [CONTRIBUTING.md](https://github.com/DGtal-team/DGtalTools/blob/master/CONTRIBUTING.md)
- [ ] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtalTools/blob/master/ChangeLog.md) added.
- [ ] Update the readme with potentially a screenshot of the tools if it applies. 
- [ ] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).
- [ ] All continuous integration tests pass (Github Actions & appveyor).

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
 # DGtalTools 1.3 (beta)
-
-
-
+- *volumetric*
+  - criticalKernelsThinning3D: new option to keep the resulting image
+  domain equals to the input image (instead using the resulting
```
Comment (dcoeurjo) [inline ChangeLog.md:4]:
```suggestion
  domain equal to the input image (instead using the resulting
```

File: volumetric/criticalKernelsThinning3D.cpp [hunk 2]
```diff
@@ -54,7 +54,7 @@
    -o,--exportImage TEXT                 Export the resulting set of points to a image compatible with GenericWriter.
    -e,--exportSDP TEXT                   Export the resulting set of points in a simple (sequence of discrete point (sdp)).
    -t,--visualize                        Visualize result in viewer
-
+   -k,--keepInputDomain                  Keep the resulting image domain equals to the input image (instead using the resulting bouding box set).
```
Comment (dcoeurjo) [inline volumetric/criticalKernelsThinning3D.cpp:57]:
```suggestion
   -k,--keepInputDomain                  Keep the resulting image domain equal to the input image (instead using the resulting bouding box set).
```

**Review by dcoeurjo — APPROVED** [review]
All good, just typos

**Comment by kerautret** [comment]
Thanks @dcoeurjo done and merged ;)

**Pull request merged**